docs/Irresponsible-Module.md

```diff
@@ -4,6 +4,10 @@
 
 Classes and modules are the units of reuse and release. It is therefore considered good practice to annotate every class and module with a brief comment outlining its responsibilities.
 
+For further guide on how to write good documentation in Ruby, see this link:
+- <http://rdoc.sourceforge.net/doc/index.html>
```

Comment:
This link is outdated. Current RDoc documentation is at https://rdoc.github.io/rdoc/. Is there a reason you prefer the version you used?

Another problem is that this is mostly a tool manual, and only touches on style as an aside.